cms/templates/sudo/sudo.html

```diff
@@ -1,7 +1,8 @@
 {% block body %}
```

Comment:
@cpennington I think we discussed it before and it wasn't possible that's why we decided not to do that.